ui/intermittents-view/BugColumn.jsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { connect } from 'react-redux';
+
+import { updateSelectedBugDetails, updateDateRange, updateTreeName } from './redux/actions';
+import { getBugUrl } from '../helpers/urlHelper';
+
+class BugColumn extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      displayBugDetails: false,
+    };
+    this.displayBugText = this.displayBugText.bind(this);
+    this.updateStateData = this.updateStateData.bind(this);
+  }
+
+  displayBugText() {
+    this.setState({ displayBugDetails: !this.state.displayBugDetails });
+  }
+
+  updateStateData() {
+    // bugdetailsview inherits data from the main view
+    const { data, updateDates, updateTree, updateBugDetails, from, to, tree } = this.props;
+    updateBugDetails(data.id, data.summary, 'BUG_DETAILS');
```

Comment:
just a nit here, but I've been told that the usual style is to separate the ``const`` declarations from the functional code by a blank line.  So between ``const`` and ``updateBugDetails`` here or ``const`` and ``return`` elsewhere.  Not a big deal though...  :)

Reply:
Ah yeah, I missed that one :)